modules/support-collecting-network-trace.adoc

```diff
@@ -25,7 +25,7 @@ When investigating potential network-related {product-title} issues, Red Hat Sup
 $ oc get nodes
 ----
 
-. Enter into a debug session on the target node. This step instantiates a debug Pod called `<node_name>-debug`:
+. Enter into a debug session on the target node. This step instantiates a debug pod called `<node_name>-debug`:
```

Comment:
Does it make more sense to say

> Start a debug session

ISG uses _enter_ for terminal input, like _Enter the following command_.